test/src/templates/big_fact_slide_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_deck/flutter_deck.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'test_utils.dart';
+
+void main() {
+  group('BigFactSlide', () {
+    testWidgets('should render all layout elements',
+        (WidgetTester tester) async {
```

Comment:
Type declaration is redundant. Please, remove it from all the tests.
```suggestion
        (tester) async {
```